applications/plugins/picopass/scenes/picopass_scene_read_card_success.c

```diff
@@ -33,7 +33,7 @@ void picopass_scene_read_card_success_on_enter(void* context) {
     uint8_t csn[PICOPASS_BLOCK_LEN];
     memcpy(csn, &AA1->data[PICOPASS_CSN_BLOCK_INDEX], PICOPASS_BLOCK_LEN);
     for(uint8_t i = 0; i < PICOPASS_BLOCK_LEN; i++) {
-        furi_string_cat_printf(csn_str, " %02X", csn[i]);
+        furi_string_cat_printf(csn_str, "%02X ", csn[i]);
```

Comment:
Nope, not going to fix issue ;-) Space is still space and it's handled when rendering. But even if you trim last space there will be not enough space.